### ISSUE TITLE
JVC RC320 Remote for SmartTV

### ISSUE THREAD

**Issue by alvarez-86** [irplus-remote/irplus-codes.github.io#520]
I tried to make a file with a RPI and lirc and I got errors when I start recording each button. Please help me get the codes

**Comment by irplus-remote** [comment]
Hi could you specify the error? Maybe we can help so you can create the correct codefile

**Comment by PelouseTech** [comment]
The first steps works fine (radonmly push buttons), but whet I start with
the button by button programming, I get a read error on every button.

**Comment by irplus-remote** [comment]
did you try to read raw codes? with "-f" parameter. You might also try the tool https://github.com/bengtmartensson/IrScrutinizer to read the codes.  

irrecord -n -f -d /dev/lirc*

**Comment by PelouseTech** [comment]
Yes, I did try the raw mode with no success, I'll Try with IrScrutinizer,
thanks for the help!
